kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PersistentVolumeClaimExample.java

```diff
@@ -39,7 +40,7 @@
   private static final Logger logger = LoggerFactory.getLogger(PersistentVolumeClaimExample.class);
 
   public static void main(String[] args) {
-    String master = "https://localhost:8443/";
+    String master = "https://192.168.99.100:8443/";
```

Comment:
I think the address should not be explicit. It's not always that with minikube or minishift. It depends. So default should stay to localhost

Reply:
ok i will changed it to localhost